src/tests/roc_pipeline/test_loopback_sink_2_source.cpp

```diff
@@ -559,6 +569,9 @@ void send_receive(int flags,
             }
         }
     }
+    if (flags & FlagInterleaving) {
+        check_metrics(*receiver_slot, *sender_slot, flags ^ FlagInterleaving);
+    }
```

Comment:
```
// While receiving interleaved packets losses could be detected incorrectly,
// so we postpone the final check for lost packets metric till the whole bunch
// of packets is sent.
if (flags & FlagInterleaving) {
    // Here we exclude FlagInterleaving from flags so that check_metrics could
    // undertake the full check.
    check_metrics(*receiver_slot, *sender_slot, flags ^ FlagInterleaving, proxy);
}
```